source/docs/software/networktables/publish-and-subscribe.rst

```diff
@@ -0,0 +1,911 @@
+Publishing and Subscribing to a Topic
+=====================================
+
+Publishing to a topic
+---------------------
+
+In order to create a :term:`topic` and publish values to it, it's necessary to create a :term:`publisher`.
+
+NetworkTable publishers are represented as type-specific Publisher objects (e.g. ``BooleanPublisher``: `Java <https://github.wpilib.org/allwpilib/docs/beta/java/edu/wpi/first/networktables/BooleanPublisher.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/beta/cpp/classnt_1_1_boolean_publisher.html>`__). The publisher object must be retained for the publisher to continue publishing. In Java, the ``close()`` method needs be called to ensure publishing is stopped; in C++ this is handled by the destructor. C++ publishers are moveable and non-copyable.
```

Comment:
This paragraph doesn't read cleanly. 

> The publisher object must be retained for the publisher to continue publishing

Can this sentence be reworded?

Reply:
Maybe "Publishers are only active as long as the Publisher object exists. Typically you want to keep publishing longer than the local scope of a function, so it's necessary to store the Publisher object somewhere longer term, e.g. in an instance variable."?